test/common/dashboards_exist.go

```diff
@@ -84,7 +84,57 @@ var rhmi2ExpectedDashboards = []dashboardsTestRule{
 	},
 }
 
-func TestIntegreatlyDashboardsExist(t *testing.T, ctx *TestingContext) {
+var customerRHOAMDashboards = []dashboardsTestRule{
+	{
+		Title: "3Scale Api Rate Limiting",
```

Comment:
@laurafitzgerald one more - does this have to be updated now as well?

Reply:
Yeah. 🙈

Reply:
Done. 👍